tsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
   "compilerOptions": {
-    "esModuleInterop": true,
-    "module": "CommonJS",
     "moduleResolution": "node",
-    "outDir": "./dist",
-    "target": "ESNext"
+    "module": "CommonJS",
+    "target": "ESNext",
+    "sourceMap": false,
+    "declaration": false,
+    "allowJs": true
   },
-  "exclude": ["node_modules", "**/*.spec.ts"],
-  "include": ["src/**/*"]
+  "exclude": ["app/**/*"]
```

Comment:
can we reinclude `"node_modules"` in the `exclude` list? it's apparently a default, but i'm not sure if specifying one ourselves adds it to the defaults or overwrites (i'm guessing the latter)